pkg/credentialprovider/aws/aws_credentials.go

```diff
@@ -30,7 +30,9 @@ import (
 	"k8s.io/kubernetes/pkg/credentialprovider"
 )
 
+const chinaRegionPrefix = "cn-"
 const registryURLTemplate = "*.dkr.ecr.%s.amazonaws.com"
+const chinaRegistryURLTemplate = "*.dkr.ecr.%s.amazonaws.com.cn"
```

Comment:
I'm sure is `amazonaws.com.cn`.

<img width="531" alt="screen shot 2017-08-04 at 5 13 49 am" src="https://user-images.githubusercontent.com/685488/28944073-b869238a-78d3-11e7-8879-395967d625c2.png">